app/controllers/applications_controller.rb

```diff
@@ -25,6 +25,11 @@ def index
     end
   end
 
+  def select
+    @event.applications.update_all(["selected = (id IN (?))", params[:selected_ids]])
+    redirect_to event_applications_path(@event), notice: "Cool! Changes saved."
+  end
```

Comment:
Please don't forget to make sure that this action is [only available to admins](https://github.com/rubymonsters/rgbapp/blob/1d7e0727fa269dbefa3dc9a78aaf14fa7fc36cff/app/controllers/applications_controller.rb#L3).